aadl.library/AADL.sysml

```diff
@@ -1,26 +1,7 @@
-/**
```

Comment:
Then, I will add mine too.
By principle, I cannot accept that our free contribution becomes copyrighted by another organization (knowing that we already agreed that it will be copyrighted by both SAE and OMG)

I would however prefer a copyleft approach for such a standardization collaborative work, otherwise it will soon become a complete mess.
A similar issue is the License file in the repository that is also irrelevant in this context.